pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/tissue/TissueSMIDDao.java

```diff
@@ -13,30 +13,26 @@
 
 import lombok.NonNull;
 import org.broadinstitute.dsm.db.SmId;
+import org.broadinstitute.dsm.exception.DsmInternalError;
 import org.broadinstitute.dsm.statics.DBConstants;
 import org.broadinstitute.lddp.db.SimpleResult;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class TissueSMIDDao {
 
-    private static final Logger logger = LoggerFactory.getLogger(TissueSMIDDao.class);
-
     public static final String SQL_GET_SM_ID_BASED_ON_TISSUE_ID =
-            " SELECT * from sm_id sm where sm.tissue_id= ?   and NOT sm.deleted <=> 1";
+            " SELECT * from sm_id sm where sm.tissue_id= ?";
     public static final String SQL_GET_SEQUENCING_SM_ID_BASED_ON_TISSUE_ID = " SELECT * from sm_id sm "
             + "left join sm_id_type smtype on (sm.sm_id_type_id = smtype.sm_id_type_id) "
             + "where sm.tissue_id = ? and (smtype.sm_id_type = \"uss\" or smtype.sm_id_type = \"scrolls\") "
-            + "and sm.received_date is not null ";
+            + "and sm.received_date is not null and NOT sm.deleted <=> 1 ";
```

Comment:
Changes added so the deleted sm ids are not included in a clinical order to Mercury